<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundLibraryOnlineImportDialog_UI</class>
 <widget class="QDialog" name="SoundLibraryOnlineImportDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="overallLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>799</width>
        <height>692</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>795</width>
        <height>688</height>
       </size>
      </property>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="TreeLayout">
             <item>
              <widget class="QComboBox" name="repositoryCombo"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="controlLayout">
               <item>
                <spacer name="lSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="Button" name="UpdateListBtn">
                 <property name="text">
                  <string>Update list</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Button" name="EditListBtn">
                 <property name="text">
                  <string>Edit server list</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="rSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTreeWidget" name="m_pDrumkitTree">
               <property name="minimumSize">
                <size>
                 <width>355</width>
                 <height>0</height>
                </size>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="indentation">
                <number>15</number>
               </property>
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="itemsExpandable">
                <bool>false</bool>
               </property>
               <column>
                <property name="text">
                 <string>1</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="propertiesLayout">
             <item>
              <widget class="QLabel" name="SoundLibraryNameLbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Sound Library Name</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="imageLayout">
               <item>
                <widget class="QTextEdit" name="SoundLibraryInfoLbl">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="drumkitImageLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>200</height>
                </size>
               </property>
               <property name="palette">
                <palette>
                 <active>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="0">
                     <red>34</red>
                     <green>31</green>
                     <blue>30</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </active>
                 <inactive>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="0">
                     <red>34</red>
                     <green>31</green>
                     <blue>30</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </inactive>
                 <disabled>
                  <colorrole role="WindowText">
                   <brush brushstyle="SolidPattern">
                    <color alpha="255">
                     <red>144</red>
                     <green>141</green>
                     <blue>139</blue>
                    </color>
                   </brush>
                  </colorrole>
                 </disabled>
                </palette>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="AuthorLbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Author...</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="LicenseLbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Drumkit License...</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="ImageLicenseLbl">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>Image License...</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="bottomLayout">
         <item>
          <spacer name="spacerLClose">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Button" name="DownloadBtn">
           <property name="text">
            <string>Download and install</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Button" name="close_btn">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spsacerRClose">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header location="global">../src/Widgets/Button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
